lib/beaker/shared/host_manager.rb

```diff
@@ -102,7 +102,7 @@ def run_block_on hosts = [], filter = nil, opts = {}, &block
         end
         if block_hosts.is_a? Array
           if block_hosts.length > 0
-            if opts[:run_in_parallel]
+            if opts[:run_in_parallel] == true
```

Comment:
The only place where it won't be a boolean is if it is used as a global option where I'm already checking to ensure that it is an array and contains specific values... the option for run_block_on is the only place anywhere in Beaker where we are checking for a true/false value, so this should be ok.